libs/front/components/src/molecules/feedback/global-indicator/GlobalIndicator.test.tsx

```diff
@@ -1,19 +1,25 @@
 import { Build } from '@mui/icons-material';
 import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
-import { render } from '../../../test';
+import { appRender } from '../../../test/renders/appRender';
 import { GlobalIndicator } from './GlobalIndicator';
 
 describe('GlobalIndicator component', () => {
+  const render = (title: string, children: string) => ({
+    user: userEvent.setup(),
+    ...appRender(
+      <GlobalIndicator hasTopMargin title={title} Icon={Build}>
+        {children}
+      </GlobalIndicator>
+    ),
+  });
```

Comment:
with the new guidelines from testing-library, usage of `appRender` becomes a bit weird

Reply:
Kinda, actually I should wrap this.

Reply:
https://github.com/jpb06/mui-rhf-sandbox/pull/18/commits/c0ab88a36d430fb234fc9c19649d2dd827a83abc